easybuild/easyconfigs/o/OrthoFinder/OrthoFinder-2.3.8-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonPackage'
+
+name = 'OrthoFinder'
+version = '2.3.8'
```

Comment:
2.3.10 was released 2 days ago, worth a bump?

Reply:
Not really, it is mainly an upgrade for the support to Python 3 and this easyconfig is for Python 2. On top of that, we have been running version `2.3.8` for a while with no issues. I suggest to wait for more releases improving the support for Python 3 and then make an easyconfig for it in `2020a`.

Reply:
OK to stick to 2.3.8 then :+1: